packages/webpack-configs/src/dev.ts

```diff
@@ -43,7 +44,7 @@ export function defineFastRefreshPluginConfig(options: ReactRefreshPluginOptions
 
 export interface DefineDevConfigOptions {
     entry?: string;
-    https?: NonNullable<WebpackConfig["devServer"]>["https"];
+    https?: boolean | ServerOptions | undefined;
```

Comment:
Could this cause a breaking change for consumers?

Reply:
No we're good, good catch thought!